### ISSUE TITLE
Cannot import plugin in another plugin from the same workspace

### ISSUE THREAD

**Issue by Jad31** [nrwl/nx#9637]
## Current Behavior
I can't import a plugin in another plugin in the same monorepo.

## Expected Behavior
I want to import a plugin in another plugin and use it in it in the same monorepo.

## Steps to Reproduce
- Clone this repository:  https://github.com/Jad31/nx-issue-import-plugin-in-another-plugin
- `yarn`
- `yarn run nx run first-plugin:build`
- `yarn run nx generate dist/libs/first-plugin:first-plugin`

### Failure Logs
```ts

> yarn run nx generate dist/libs/first-plugin:first-plugin                
yarn run v1.22.11
$ workspace-poc/node_modules/.bin/nx generate dist/libs/first-plugin:first-plugin
Unable to resolve dist/libs/first-plugin:first-plugin.
Cannot find module '@workspace-poc/second-plugin'
Require stack:
- /home/ash/github.com/workspace-poc/dist/libs/first-plugin/src/generators/first-plugin/generator.js
- /home/ash/github.com/workspace-poc/node_modules/nx/src/shared/workspace.js
- /home/ash/github.com/workspace-poc/node_modules/nx/src/cli/init-local.js
- /home/ash/github.com/workspace-poc/node_modules/nx/bin/nx.js
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```
### Environment
```bash

> yarn run nx report
yarn run v1.22.11
$ workspace-poc/node_modules/.bin/nx report

 >  NX   Report complete - copy this into the issue template

   Node : 14.19.1
   OS   : linux x64
   yarn : 1.22.11
   
   nx : 13.9.5
   @nrwl/angular : Not Found
   @nrwl/cypress : Not Found
   @nrwl/detox : Not Found
   @nrwl/devkit : 13.9.5
   @nrwl/eslint-plugin-nx : 13.9.5
   @nrwl/express : Not Found
   @nrwl/jest : 13.9.5
   @nrwl/js : 13.9.5
   @nrwl/linter : 13.9.5
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : Not Found
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : 13.9.5
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : Not Found
   @nrwl/web : Not Found
   @nrwl/workspace : 13.9.5
   typescript : 4.5.5
   rxjs : 6.6.7
   ---------------------------------------
   Community plugins:

Done in 0.54s.

```

### Related issues

- #6712

### Notes

@AgentEnder I saw that you worked on this issue #9116 but it seems that it doesnt fix my problem. Have you an idea of what i am doing wrong ?

**Comment by AgentEnder** [comment]
Try latest betas and refer to the plugin by it's import path, rather than the dist. This should be fixed, but 13.10s not out yet.

**Comment by AgentEnder** [comment]
This has been released in 13.10, and w/ 14 now out I'm going to close the issue out. Please open a new issue if you are still having issues. Thanks!
